sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java

```diff
@@ -46,6 +49,21 @@ public void logIn() {
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[type='submit']"))).click();
     }
 
+    public boolean logInAADConditionalAccess() {
+        driver.get(app.root() + "oauth2/authorization/azure");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.name("loginfmt"))).sendKeys(username + Keys.ENTER);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.name("passwd"))).sendKeys(password + Keys.ENTER);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[type='submit']"))).click();
+        driver.get((app.root() + "obo"));
+        try {
+            Thread.sleep(30000);
+        } catch (InterruptedException e) {
+            LOGGER.error("Current Thread Failed to sleep", e);
+        }
+        this.httpGet("obo");
+        return driver.findElement(By.tagName("body")).getText().contains("response success");
```

Comment:
I think it's better here in the helper class only returns the response, and let the test case decide whether the response matches an expected value.

Reply:
Thanks, I will do it.